Lib/zipimport.py

```diff
@@ -353,17 +353,43 @@ def _read_directory(archive):
         raise ZipImportError(f"can't open Zip file: {archive!r}", path=archive)
 
     with fp:
+        end_cent_dir_size = 22
```

Comment:
Perhaps make this a module constant, all upper cased?  Also, what does the abbreviation "cent" mean?

Reply:
Done. "cent" means "central".